DominicGBauer/information-access
# fix: Redirect bug

# Pull Request

## Scope
Fixed redirect issue with netlify. 404 page occurs on refresh.

## Work Done
added in redirects  see link (https://medium.com/@ishoshot/page-not-found-on-reload-vuejs-netlify-c71716e97e6) for article on 404 page on reset for netlify.

## Steps to Test

See https://develop.aaronskit.org/ and go to FAQ or About and refresh with button. If it refreshes to show same page then the bug is fixed.

## Screenshot
![image](https://user-images.githubusercontent.com/80747408/138010572-e8894527-0382-4f48-8dd2-6dae27760dac.png)
